File: Assets/Plugins/VoxelBusters/EssentialKit/Plugins/iOS/BillingServices/Modules/BillingServices/Enums/BillingProductSubscriptionPriceIncreaseStatus.swift
//
//  BillingProductSubscriptionPriceIncreaseStatus.swift
//  BillingServices
//
//  Created by Ayyappa on 04/04/24.
//

import Foundation

@objc
public enum BillingProductSubscriptionPriceIncreaseStatus: Int {
    case unknown
    case noIncreasePending
    case agreed
    case pending
}

File: Assets/Plugins/VoxelBusters/EssentialKit/Runtime/Core/BillingServices/BillingProductSubscriptionPriceIncreaseStatus.cs
namespace VoxelBusters.EssentialKit
{
    /// <summary>
    /// Status of billing product subscription price increase action
    /// </summary>
    public enum BillingProductSubscriptionPriceIncreaseStatus : int
    {
        /// <summary>
        /// Unknown status of billing product subscription price increase action
        /// </summary>
        Unknown,
        /// <summary>
        /// No price increase is pending
        /// </summary>
        NoIncreasePending,
        /// <summary>
        /// User agreed to the price increase
        /// </summary>
        Agreed,
        /// <summary>
        /// Price increase is pending
        /// </summary>
        Pending
    }
}